eta-company-calendar-backend/src/main/java/hu/flowacademy/companycalendar/exception/GlobalExceptionHandler.java

```diff
@@ -0,0 +1,26 @@
+package hu.flowacademy.companycalendar.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+@ResponseBody
+public class GlobalExceptionHandler {
+
+  @ExceptionHandler
+  @ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
Nem muszáj megadni, mivel a visszatérési értékünk is ez. Vagy legyen void a metódus, vagy vegyük le az annotációt.

Reply:
leveszem az annotációt